.github/workflows/test.yml

```diff
@@ -89,6 +92,7 @@ jobs:
         python -m pip install --upgrade pip
         pip install pytest
         pip install -r test_requirements/${{ matrix.requirements-file }}
+        pip install -r test_requirements/mysql.txt
```

Comment:
Installing mysqlclient and psycopg2 doesn't in base is fine IMO.

Reply:
Can you please revert this changes & put mysqlclient and psycopg2 back in requirements_base.txt file?